.rubocop.yml

```diff
@@ -1,16 +1,503 @@
 require:
   - rubocop-performance
-  - rubocop-rspec
   - rubocop-rails
+  - rubocop-rspec
+
+inherit_from: .rubocop_todo.yml
 
 AllCops:
   TargetRubyVersion: 3.2
   DisplayCopNames: true
   SuggestExtensions: false
-  NewCops: disable
```

Comment:
if we don't disable these, then rubocop errors can overwhelm the FR.

Reply:
@ndushay They're disabled by default.